src/assets/custom/css/_sass/_soft-mod.scss

```
@@ -84,7 +89,24 @@
   }
   @media (min-width: 992px) and (max-width: $largeMax) {
     .page-location-anchor {
-      top: calc(40px - #{$site-navbar-height-large});
+      top: calc(40px - #{$site-navbar-height-distorted});
     }
   }
 }
+
+.soft-mod__get-in-touch-form {
+  @include max-width--column;
+  background-color: white;
+  padding-bottom: 100px;
+
+  @include small {
+    padding-top: 50px;
+    padding-left: 15px;
+    padding-right: 15px;
+  }
+
+  @include medium-large-and-extra-large {
+    margin-top: -150px;
+    padding-top: 100px;
```

We could use the overlap variable to explain these two values.
```suggestion
    margin-top: #{-1.5 * $vertical-overlap};
    padding-top: $vertical-overlap;
```